pkg/ddl/reorg.go

```diff
@@ -212,8 +212,12 @@ func (rc *reorgCtx) getRowCount() int64 {
 // the additional ddl round.
 //
 // After that, we can make sure that the worker goroutine is correctly shut down.
-func (w *worker) runReorgJob(reorgInfo *reorgInfo, tblInfo *model.TableInfo,
-	lease time.Duration, f func() error) error {
+func (w *worker) runReorgJob(
+	reorgInfo *reorgInfo,
+	tblInfo *model.TableInfo,
+	lease time.Duration,
+	reorgJob func() error,
```

Comment:
```suggestion
	reorgFn func() error,
```
reorgJob sounds like a `job`